src/components/StyleGuide/ContainerQuery.mdx

```diff
@@ -0,0 +1,250 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import Link from "../Link/Link";
+
+<Meta title="Development Guide/Container Query Guide" />
+
+# Container Query Guide
+
+## Table of Contents
+
+- {<Link href="#general-information" target="_self">General Information</Link>}
+- {<Link href="#updating-a-component-to-use-container-queries" target="_self">Updating a component to use container queries</Link>}
+- {<Link href="#full-componentexample" target="_self">Full component example</Link>}
+
+The `setContainerStyles` function helps you generate container query styles based
```

Comment:
This is helpful but I think it can be moved down below to the `setContainerStyles` section, unless if you did want to mention it in this intro?

Reply:
Or, perhaps, it can go at the end of general information. That section sets up that this is a guide and what container queries are, and then this paragraph further shows what is coming up in this guide to help write container queries in the DS.

Reply:
I forgot to move this after I added the other sections. Moved to the end of general information.